docs/dashboard/README.md

```diff
@@ -29,3 +29,26 @@ Currently, the Serverless Framework Dashboard supports the following providers,
 - Regions: us-east-1, us-east-2, us-west-2, eu-central-1, eu-west-1, eu-west-2, ap-northeast-1, ap-south-1, ap-southeast-1, ap-southeast-2
 
 Need unsupported providers, runtimes or regions? [Let us know](https://serverless.com/enterprise/) what you need. We are working hard to quickly expand our supported runtimes and regions.
+
+# Installing
+
+To get started with the Serverless Framework Dashboard, follow the [Getting Started with the Serverless Framework and AWS](/framework/docs/getting-started/) guide. When you run the `serverless` command, you will be asked if you would like to enable dashboard features.
+
+# Enabling the Dashboard on existing Serverless Framework services
+
+If you have an existing Serverless Framework service, it is incredibly easy to enable the Serverless Framework Dashboard features. Just follow the [Getting Started with the Serverless Framework and AWS](/framework/docs/getting-started/) guide to install update the Serverless Framework to the latest release.
+
+If you don't already have a Serverless account, create a new account at [https://dashboard.serverless.com].
```

Comment:
[https://dashboard.serverless.com]([https://dashboard.serverless.com])

Reply:
This includes a subset of the other changes with some tweaks. This one will go live now. The other one soon after, once we launch the CI/CD features.